sklearn/manifold/_mds.py

```diff
@@ -426,11 +502,21 @@ def fit_transform(self, X, y=None, init=None):
             raise ValueError("Proximity must be 'precomputed' or 'euclidean'."
                              " Got %s instead" % str(self.dissimilarity))
 
-        self.embedding_, self.stress_, self.n_iter_ = smacof(
-            self.dissimilarity_matrix_, metric=self.metric,
-            n_components=self.n_components, init=init, n_init=self.n_init,
-            n_jobs=self.n_jobs, max_iter=self.max_iter, verbose=self.verbose,
-            eps=self.eps, random_state=self.random_state,
-            return_n_iter=True)
+        if self.method == "smacof":
+            self.embedding_, self.stress_, self.n_iter_ = smacof(
```

Comment:
Please update the doc of the n_iter_ attribute to indicate that it is `None` when method=svd (and actually set it to None in the code ;) )

Reply:
Done